website/content/en/docs/concepts/policies.md

```diff
@@ -70,10 +68,19 @@ Recall that a connection is dropped if it does not match any access policy.
  must be created on both sides of the connection.
  Creating an access policy is accomplished by creating an `AccessPolicy` CR in
  the relevant namespace (see Note above).
+ Creating a high-priority access policy is accomplished by creating a `PrivilegedAccessPolicy` CR.
+ Instances of `PrivilegedAccessPolicy` have no namespace and affect the entire cluster.
 
 {{% expand summary="Export Custom Resource" %}}
```

Comment:
The summary should have `Privileged Access Policy Custom Resource`, not `Export`